addons/point_of_sale/static/src/app/models/related_models.js

```diff
@@ -164,6 +164,10 @@ export class Base {
         }
     }
 
+    getDefaultVals() {
```

Comment:
would it make sense in your opinion to declare the default values in python ?

Reply:
I was actually thinking of doing that, putting default values on the required fields. And maybe, we can fetch those default primitive values.

Reply:
indeed, then in the model files you wouldn't need to do anything. You could just add the defaults in the create inside related_models, right ?